.rubocop.yml

```diff
@@ -1,9 +1,10 @@
 inherit_from:
-  - https://raw.githubusercontent.com/carbonfive/c5-conventions/master/rubocop/rubocop.yml
+  - https://raw.githubusercontent.com/carbonfive/c5-conventions/main/rubocop/.rubocop-common.yml
+  - https://raw.githubusercontent.com/carbonfive/c5-conventions/main/rubocop/.rubocop-performance.yml
+  - https://raw.githubusercontent.com/carbonfive/c5-conventions/main/rubocop/.rubocop-rails.yml
+  - https://raw.githubusercontent.com/carbonfive/c5-conventions/main/rubocop/.rubocop-rspec.yml
 
 require:
   - rubocop-performance
   - rubocop-rails
-
-Rails:
-  Enabled: true
```

Comment:
This is not needed. If we require `rubocop-rails` then the Rails rules are implicitly enabled. This syntax was used back when the Rails rules were included within rubocop itself. Now they are a separate opt-in gem.